migrations/m150623_212711_fix_username_notnull.php

```diff
@@ -36,7 +36,7 @@ public function down()
             if ($this->dbType == 'sqlsrv') {
                 $this->dropIndex('{{%user_unique_username}}', '{{%user}}');
             }
-            $this->alterColumn('{{%user}}', 'username', $this->string(255)->null());
+            $this->alterColumn('{{%user}}', 'username', $this->string(255), ' NULL');
```

Comment:
For some strange reasons SQL server does not accept the ->null(), it creates a strange SQL like
DEFAULT NULL NULL string which is not correct